packages/flutter_tools/lib/src/devtools_launcher.dart

```diff
@@ -5,12 +5,14 @@
 import 'dart:async';
 
 import 'package:browser_launcher/browser_launcher.dart';
+import 'package:http/http.dart' as http;
 import 'package:meta/meta.dart';
 import 'package:process/process.dart';
 
 import 'base/io.dart' as io;
 import 'base/logger.dart';
 import 'convert.dart';
+import 'globals.dart' as globals;
```

Comment:
Instead of using `globals`, provide `PersistentToolState` in the constructor of the devtools launcher

Reply:
done